### DISCUSSION TITLE
React · 검색 기능에 Debounce 적용하기 | thisyujeong.dev

### DISCUSSION THREAD

**Discussion by giscus** [thisyujeong/thisyujeong-dev#16] [General]
# React · 검색 기능에 Debounce 적용하기 | thisyujeong.dev

feat. debounce와 throttle의 차이

https://thisyujeong.dev/blog/react-debounce

**Comment by fanduck-com** [comment]
잘보고갑니다! Debounce 되게 유용한데  사용하다가 저는 그냥 Debounce 안쓰고 일반적인 검색과정처럼 search 페이지에 query를 넘겨서 해당 페이지에서 검색하도록 변경했습니다! 개인적으로 전 이게 더 좋더라구욤 ㅋㅋ

**Reply by thisyujeong** [reply]
로이더님! 이렇게 포스팅 댓글에서 뵈니 더 반가워요~ query를 넘겨서 구현하는 것도 편하죠! 
검색을 구현하는 것만 해도 무수히 많은 방법이 있다니 공부하면 할수록 재미있지 않나요! 😁😁😁

**Reply by fanduck-com** [reply]
맞아요~ 덕분에 많이 배워서 현재 블로그는 완전 땅에서부터 헤딩해서 만들어봤습니다! 도움이 많이 컸어요 😊😁 자주 소통해요~~
